lib/publiccloud/instance.pm

```diff
@@ -46,8 +47,10 @@ sub run_ssh_command {
 
     my $ssh_cmd = sprintf('ssh %s -i "%s" "%s@%s" -- %s',
         $self->ssh_opts, $self->ssh_key, $self->username, $self->public_ip, $args{cmd});
-    record_info('CMD', $ssh_cmd);
-    return script_output($ssh_cmd, $args{timeout}, %args);
+    record_info('SSH CMD', $ssh_cmd);
+    delete($args{cmd});
```

Comment:
I fail to find usage of `cmd` inside `script_output` , did I miss something  ? anyway what is your motivation to delete it ?

Reply:
just, avoid passing confusing arguments to a function.